### PR TITLE
Enable ChangeLog verification using updated docWarden

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Switch to the latest version of Doc-Warden
- Verify changelogs (presence and content)
- Verify release note entry on manually queued builds

### CODE REVIEW

File: eng/pipelines/templates/steps/analyze.yml [hunk 1]
```diff
@@ -10,8 +10,15 @@ steps:
 
   - script: |
      pip install -r eng/ci_tools.txt
-     ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/.docsettings.yml
-    displayName: 'Verify Readmes'
+     ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/.docsettings.yml -t all
```
Comment (weshaggard) [inline eng/pipelines/templates/steps/analyze.yml]: Do we want to do this now or after we enable the changelog update as part of version increment? I guess for python they already validate the changelog so presumably this wouldn't add any extra burden, but we should switch off the other validation now that we are using doc-warden.

**Comment by chidozieononiwu** [comment]
Closing this as changelog tooling is now in azure-sdk-tools

**Comment by weshaggard** [comment]
>Closing this as changelog tooling is now in azure-sdk-tools

Do we still have the issue tracking the validation of the changelogs?

**Comment by chidozieononiwu** [comment]
> > Closing this as changelog tooling is now in azure-sdk-tools
> 
> Do we still have the issue tracking the validation of the changelogs?

I created one to track that https://github.com/Azure/azure-sdk-tools/issues/421

**Pull request closed without merging**